easybuild/easyconfigs/r/RStudio-Server/RStudio-Server-2022.02.0-443-rhel-x86_64.eb

```diff
@@ -0,0 +1,37 @@
+easyblock = 'Binary'
+
+name = 'RStudio-Server'
+version = '2022.02.0-443'
+versionsuffix = '-rhel-%s' % ARCH
+
+homepage = 'https://www.rstudio.com/'
+description = """This is the RStudio Server version.
+RStudio is a set of integrated tools designed to help you be more productive with R.
+
+The server can be started with:
+  rserver --server-daemonize=0 --www-port 8787 --rsession-which-r=$(which R)
+"""
+
+toolchain = SYSTEM
```

Comment:
Any particular reason to push RStudio-Server to the SYSTEM toolchain and perform it as a binary install, instead of build from sources with `foss` (like all of the previous EasyConfigs)?

Reply:
Rstudio server is a webservice, and we offer it to the users so they can load any R version they want with it. RStudio server itself doesn't need R, but obviously it's useless without it. 

any toolchain dependency is thus more harmful in the way we want to use. we could work around it, but i see very little advantage in a source build (esp if there are rpms targeted for the OS we use).

i am aware this is not very EB-like, but otoh, it does work. the easyconfig could be renmaed to `rhel8` instead of just rhel (@boegel was this intentional?)

Reply:
This is a problem for EESSI though, where we will actually need the source build

Reply:
(Of course we could have both)

Reply:
@ocaisa i'm also quite sure we can refactor our wrappers and be able to load the rstudio independent of the R module. so for EESSI this is rpm approach is not required.
 but using an rpm was easier and less work